lib/command.js

```diff
@@ -228,15 +229,18 @@ module.exports = function (proto) {
 
     if (timeout) {
       timeoutId = setTimeout(function(){
-        err = new Error('gm() resulted in a timeout.');
-        cb(err);
-        if (proc.connected) {
```

Comment:
this check was useless because `child_process.connected` is always `false` without forked process or not using ipc.
( please looks arround here https://github.com/nodejs/node/blob/master/lib/internal/child_process.js#L337)
So, I change this line to check `proc.exitCode` at https://github.com/aheckmann/gm/pull/487/files#diff-b26080cc6f2472dce354dbedffc64d57R329